docs/jsx.md

```diff
@@ -183,6 +184,34 @@ You can use hooks in your production environment to run the production build scr
 }
 ```
 
+#### Making `m` accessible globally
+In order to access `m` globally from all your project first import `webpack` in your `webpack.config.js` like this:
+```
+const webpack = require('webpack')
+```
+Then create a new plugin in the `plugins` property of the Webpack configuration object:
+```js
+{
+	plugins: [
+		new webpack.ProvidePlugin({
+			m: 'mithril'
+		})
+	]
+}
+```
+See [the Webpack docs](https://webpack.js.org/plugins/provide-plugin/) for more information on `ProvidePlugin`.
+
+---
+
+### Differences with React
+JSX in Mithril has some subtle but important differences compared to React's JSX.
+
+#### `class` vs `className`
+React follows a purist JavaScript approach hence `class` is a reserved word that cannot be used in JSX. In Mithril both are valid since we believe `class` is more idiomatic to HTML markup.
+
+#### DOM events
+Mithril uses the native syntax for binding DOM events. Instead of using React's `onClick` or `onSubmit` syntax (which are handlers for its synthetic event system) you'd use the native `onclick` or `onsubmit`. Mithril supports all DOM element event bindings.
```

Comment:
This section is a bit incomplete, is somewhat inaccurate, and for the `class` vs `className` part in particular, is unnecessarily biased. For one concrete example, the attribute issue is not unique to just `class` vs `className` - there's others like `for` vs `htmlFor`, and it extends to both attributes and style properties.

This aligns closer to the truth:

```md
#### Attribute and style property case conventions
React requires you use the camel-cased DOM property names instead of HTML attribute names for all attributes other than `data-*` and `aria-*` attributes, using `className` instead of `class` and `htmlFor` instead of `for`, for example. In Mithril, it's more idiomatic to use the lowercase HTML attribute names instead, as Mithril always falls back to setting attributes if a property doesn't exist and it aligns more intuitively with HTML. Note that in most cases, the DOM property and HTML attribute names are either the same or very similar, such as `value`/`checked` for inputs and the `tabindex` global attribute vs the `elem.tabIndex` property on HTML elements. Very rarely do they differ beyond case: the `elem.className` property for the `class` attribute or the `elem.htmlFor` property for the `for` attribute are among the few exceptions.

Similarly, React always uses the camel-cased style property names exposed in the DOM via properties of `elem.style` (like `cssHeight` and `backgroundColor`), while Mithril supports both that and the kebab-cased CSS property names (like `height` and `background-color`) and idiomatically prefers the latter. Only `cssHeight`, `cssFloat`, and vendor-prefixed properties differ in more than case.

#### DOM events
React upper-cases the first character of all event handlers: `onClick` listens for `click` events and `onSubmit` for `submit` events. Some are further altered as they're multiple words concatenated together: `onMouseMove` listens for `mousemove` events, for instance. Mithril does not do this case mapping, but instead just prepends `on` to the native event, so you'd add listeners for `onclick` and `onmousemove` to listen to those two events, respectively. This corresponds much more closely to HTML's naming scheme and is much more intuitive if you come from an HTML or vanilla DOM background.

React supports scheduling event listeners during the capture phase (in the first pass, out to in, as opposed to the default bubble phase going in to out in the second pass) by appending `Capture` to that event. Mithril currently lacks such functionality, but it could gain this in the future. If this is necessary, you can manually add and remove your own listener in [lifecycle hooks](lifecycle.md).
```

Reply:
Totally agree.

I've made a commit using your text. I did minor style changes to simplify some of the sentences and corrected the use of some commas.